app/validators/sections/section_validator.py

```diff
@@ -0,0 +1,255 @@
+from collections import defaultdict
+
+from app import error_messages
+from app.validators.validator import Validator
+from app.validators.questions import get_question_validator
+from app.validators.routing.answer_routing_validator import AnswerRoutingValidator
+from app.validators.routing.routing_validator import RoutingValidator
+from app.validators.routing.when_rule_validator import WhenRuleValidator
+from app.validators.answers import get_answer_validator
+from app.validators.blocks import get_block_validator
+
+
+class SectionValidator(Validator):
+    QUESTIONNAIRE_MUST_CONTAIN_PAGE = (
+        "Questionnaire must contain one of [Confirmation page, Summary page, Hub page]"
+    )
+    QUESTIONNAIRE_ONLY_ONE_PAGE = "Questionnaire can only contain one of [Confirmation page, Summary page, Hub page]"
```

Comment:
These all provide a way to submit, therefore I think we should use `SUBMISSION_PAGE` instead of `PAGE` for clarity.